app/controllers/statistics_controller.rb

```diff
@@ -38,8 +39,31 @@ def for_groups
   end
 
   def for_users
+    zip_data = Zip::OutputStream.write_buffer do |zip|
```

Comment:
Biztos indokolt a három különálló export elkészítése? Nem lenne elég egy CSV-be beleírni az összes adatot?
Mondjuk lehetne az exportban 4 mező:

- Név
- Saját rendelés nyomtatás
- Mások rendeléseinek nyomtatása 
- Összesen nyomtatva

Szerintem igy könyebb lenne átlátni.

Reply:
Eleinte én is így csináltam, hogy több oszlopot adtam hozzá, viszont így nem átlátható, hogy milyen rendelésszámok voltak saját / nem saját nyomtatásban.

Pl.: van egy embernek 10-10 pontja saját és nem saját rendelésekből. Az egyiket 80 db matricával, a másikat 10 db A4-es rendeléssel szerezte. A 80 db matricával többet dolgozott mint a 10 db A4-es papírral, így közipontot nagyobb súllyal adok a matricás rendelésekre.

Ha egybe van ömlesztve, akkor nem látom a rendelésszámokat külön-külön és nehezebben tudom elosztani a közipontokat.

A saját / nem saját rendelések két külön közipont elv, így mindkettőre lehet munkapontot kapni, amit aztán még plussz képletekkel közipontokká varázsolok.

Elnézést, az elvek kicsit nagyon bonyolultra sikerültek, meg a munkapont - közipont átváltás is. Azért ilyen a rendszer, mert nem mindenki egyforma idő alatt vágja fel a rendeléseket, viszont ki kell jönnie nagy átlagban a 3 óra / 1 közipontnak

Reply:
Rendicsek, akkor ez maradhat igy, ha ez az opitmális módja.